src/graphql/admin/schema.graphql

```diff
@@ -349,6 +339,17 @@ scalar SafeInt
 """
 scalar SatAmount
 
+"""
+Price amount expressed in base/offset. To calculate, use: `base / 10^offset`
+"""
+type SettlementPrice {
+  base: SafeInt!
+  currency: DisplayCurrency!
```

Comment:
should probably also have the underlying currency, USD or BTC dpendanding of the wallet used.

Reply:
why? we already have `settlementCurrency`